WcaOnRails/app/views/teams/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<% provide(:title, 'Teams') %>
+
+<div class="container">
+  <h1><%= yield(:title) %></h1>
+
+  <% can_edit = current_user.can_edit_users? %>
+  <% if can_edit %>
+    <%= link_to "New team", new_team_path %>
+  <% end %>
+
+  <% if @teams.empty? %>
+    <div class="alert alert-warning">There are currently no teams.</div>
+  <% else %>
+    <table class="table">
```

Comment:
We could, but I don't like the look:
![wca_table](https://cloud.githubusercontent.com/assets/1881933/13462884/5dd46092-e068-11e5-9af9-f419faf3409f.png)